package.json

```diff
@@ -30,16 +30,21 @@
     "ember-cli": "^2.4.2",
     "ember-cli-blueprint-test-helpers": "^0.12.0",
     "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-internal-test-helpers": "^0.7.0",
+    "ember-cli-is-package-missing": "^1.0.0",
+    "ember-router-generator": "^1.2.1",
```

Comment:
hmm... that's surprising since it seems to run fine without them on Travis and locally here. you haven't modified any code or required them anywhere so you shouldn't need to add them. is it possible that some other dependencies on your machine were not up-to-date?